fdbclient/DatabaseBackupAgent.actor.cpp

```diff
@@ -670,74 +668,64 @@ namespace dbBackup {
 			state int nRanges = ranges.size();
 
-			// TraceEvent(SevInfo, "debug.CopyLogRangeTaskFunc::_execute.10").detail("beginVersion", beginVersion).detail("endVersion", endVersion).detail("newEndVersion", newEndVersion).detail("nRanges", nRanges);
-
 			state std::vector<PromiseStream<RCGroup>> results;
 			state std::vector<Future<Void>> rc;
 			state std::vector<Reference<FlowLock>> locks;
 			state Version nextVersion = beginVersion;
 			state double breakTime = timer_monotonic() + CLIENT_KNOBS->COPY_LOG_TASK_DURATION_NANOS;
 			state int rangeN = 0;
-			
+
 			loop {
-			  if (rangeN >= nRanges)
-			    break;
-
-			  // prefetch
-			  int prefetchTo = std::min(rangeN + CLIENT_KNOBS->COPY_LOG_PREFETCH_BLOCKS, nRanges);
-			  
-			  for (int j = results.size(); j < prefetchTo; j ++) {
-			    results.push_back(PromiseStream<RCGroup>());
-			    locks.push_back(Reference<FlowLock>(new FlowLock(CLIENT_KNOBS->BACKUP_LOCK_BYTES)));
-			    rc.push_back(readCommitted(taskBucket->src, results[j], Future<Void>(Void()), locks[j], ranges[j], decodeBKMutationLogKey, true, true, true));
-			  }
-
-			  // copy the range
-			  Optional<Version> nextVersionBr = wait(
-			    dumpData(
-			      cx, task, results[rangeN], locks[rangeN].getPtr(), taskBucket, breakTime
-			    )
-			  );
-
-			  // exit from the task if a timeout occures
-			  if (nextVersionBr.present()) {
-			    nextVersion = nextVersionBr.get();
-			    // cancel prefetch
-			    TraceEvent(
-			      SevInfo, 
-			      "CopyLogRangeTaskFunc is broken on timeout")
-				.detail("durationNanos", CLIENT_KNOBS->COPY_LOG_TASK_DURATION_NANOS)
-				.detail("rangeN", rangeN)
-				.detail("bytesWritten", Params.bytesWritten().getOrDefault(task)
-			    );
-			    for (int j = results.size(); --j >= rangeN;)
-			      rc[j].cancel();
-			    break;
-			  }
-			  // the whole range has been dumped
-			  nextVersion = getLogKeyVersion(ranges[rangeN].end);
-			  rangeN ++;
+				if (rangeN >= nRanges)
+					break;
+
+				// prefetch
+				int prefetchTo = std::min(rangeN + CLIENT_KNOBS->COPY_LOG_PREFETCH_BLOCKS, nRanges);
+
+				for (int j = results.size(); j < prefetchTo; j ++) {
+					results.push_back(PromiseStream<RCGroup>());
+					locks.push_back(Reference<FlowLock>(new FlowLock(CLIENT_KNOBS->COPY_LOG_READ_AHEAD_BYTES)));
+					rc.push_back(readCommitted(taskBucket->src, results[j], Future<Void>(Void()), locks[j], ranges[j], decodeBKMutationLogKey, true, true, true));
+				}
+
+				// copy the range
+				Optional<Version> nextVersionBr = wait(
+					dumpData(
+						cx, task, results[rangeN], locks[rangeN].getPtr(), taskBucket, breakTime
+					)
+				);
+
+				// exit from the task if a timeout occurs
+				if (nextVersionBr.present()) {
+					nextVersion = nextVersionBr.get();
+					// cancel prefetch
+					TraceEvent(
+						SevInfo, 
+						"CopyLogRangeTaskFunc is broken on timeout")
```

Comment:
```suggestion
						"CopyLogRangeTaskFuncAborted")
```
We usually don't put space in the event name.